tests/requests_spec.lua

```diff
@@ -551,4 +571,61 @@ describe("Lsp request", function()
       },
     })
   end)
+
+  it(
```

Comment:
Maybe we want put this test in different file because they are a bit different they more testing behavriour than raw data like rest of them.

Reply:
moved